src/chainparams.cpp

```diff
@@ -367,6 +370,9 @@ class CRegTestParams : public CTestNetParams
         nBlockFirstFraudulent = 999999999; //First block that bad serials emerged
         nBlockLastGoodCheckpoint = 999999999; //Last valid accumulator checkpoint
 
+        // Fake Serial Attack
+        nFakeSerialBlockheightEnd = 0;
```

Comment:
This needs to be `-1` to avoid an unnecessary recalculation of non-existent accumulators and zerocoin balances when connecting block 1 in the chain. `ConnectBlock()`'s use of this adds a static `+ 1` to the `nHeight`